baseplate/integration/pyramid.py

```diff
@@ -55,8 +55,9 @@ class BaseplateConfigurator(object):
 
     :param baseplate.core.Baseplate baseplate: The Baseplate instance for your
         application.
-    :param bool trust_trace_headers: Should this app trust trace headers from
-        the client? If ``False``, trace IDs will be generated for each request.
+    :param bool trust_trace_headers: Should this application trust trace
+        headers from the client? If ``False``, trace IDs will be generated for
```

Comment:
Maybe explain the True case? 
"If True, trace headers from the client are used. If False, trace IDs will be generated for each request."
As with Greg's comments generally - if this already makes sense to someone more acquainted with the relevant libraries then maybe it doesn't need the more explicit comment. Given my unfamiliarity, I had to read through it a couple of times to understand what happens here.